Framework/Algorithms/src/CrossCorrelate.cpp

```diff
@@ -135,6 +135,9 @@ void CrossCorrelate::exec() {
   // Create a 2DWorkspace that will hold the result
   const int nY = static_cast<int>(refY.size());
   const int npoints = 2 * nY - 3;
+  if (npoints < 1)
```

Comment:
Yes, I've added a unit test for this as part of [this branch](https://github.com/mantidproject/mantid/tree/gemmaguest/18215_add_test_for_CrossCorrelate), which I'll submit as a separate PR.